tests/test_cameraMapper.py

```diff
@@ -420,7 +420,7 @@ def testStandardizeFiltersFilterDefs(self):
             (None, None),
             (afwImage.FilterLabel(band="i", physical="i.MP9701"),
              afwImage.FilterLabel(band="i", physical="i.MP9701")),
-            (afwImage.FilterLabel(band="i"), afwImage.FilterLabel(band="r", physical="i.MP9701")),
+            (afwImage.FilterLabel(band="i"), afwImage.FilterLabel(band="i", physical="i.MP9701")),
```

Comment:
Why this change to remove a `band=r` label? Everything in here is now `band=i`, which seems a bit odd and lacking in coverage, maybe?

Reply:
The second element of the tuple is the desired _output_, and `band="r", physical="i.MP9701"` should never be the right answer. I'm a bit disturbed it didn't fail before; I think that means it got eliminated by all the "here's what you can actually determine" logic below this point.

Reply:
Would you mind adding a comment that these tuples are "input/output"? Yeah, it's gen2, but still...